resources/public/api/conf/P1.0/sandbox/income-sa-employments-request.md

```diff
@@ -37,6 +37,13 @@
          <p>{ &quot;code&quot; : &quot;INVALID_REQUEST&quot;,<br/>&quot;message&quot; : &quot;Invalid time period requested&quot; }</p></td>
     </tr>
     <tr>
+         <td><p>toTaxYear later than the current tax year</p></td>
+         <td><p>Any valid tax year where toTaxYear is later than the current tax year</p>
```

Comment:
That does not sound right "Any valid tax year where toTaxYear is later than the current tax year"
Shouldnt we say instead: "toTaxYear later than the current tax year"
And the one about should probably be "toTaxYear earlier than fromTaxYear"

Reply:
👍